<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DIDoctor</class>
 <widget class="QDialog" name="DIDoctor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="19" column="1">
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>科室编号：</string>
          </property>
         </widget>
        </item>
        <item row="23" column="3">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="2" colspan="3">
         <widget class="QLineEdit" name="leIono"/>
        </item>
        <item row="8" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>医生姓名：</string>
          </property>
         </widget>
        </item>
        <item row="11" column="2" colspan="3">
         <widget class="QLineEdit" name="leIdage"/>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>工作编号：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="3">
         <widget class="QLineEdit" name="leIdwcno"/>
        </item>
        <item row="14" column="2" colspan="3">
         <widget class="QLineEdit" name="leIdsex"/>
        </item>
        <item row="17" column="2" colspan="3">
         <widget class="QLineEdit" name="leIdtitle"/>
        </item>
        <item row="20" column="1">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>工资信息：</string>
          </property>
         </widget>
        </item>
        <item row="20" column="2" colspan="3">
         <widget class="QLineEdit" name="leIdsalary"/>
        </item>
        <item row="22" column="1">
         <spacer name="verticalSpacer_8">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="23" column="1" colspan="2">
         <widget class="QPushButton" name="btnIDoctor">
          <property name="text">
           <string>下一个</string>
          </property>
         </widget>
        </item>
        <item row="23" column="4">
         <widget class="QPushButton" name="btnIClose">
          <property name="text">
           <string>关闭</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>医生年龄：</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2" colspan="3">
         <widget class="QLineEdit" name="leIdname"/>
        </item>
        <item row="14" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>医生性别：</string>
          </property>
         </widget>
        </item>
        <item row="17" column="1">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>职称：</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <spacer name="verticalSpacer_14">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <spacer name="verticalSpacer_11">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="9" column="1">
         <spacer name="verticalSpacer_13">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="25" column="1">
         <spacer name="verticalSpacer_9">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="1">
         <spacer name="verticalSpacer_12">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <spacer name="verticalSpacer_10">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="18" column="1">
         <spacer name="verticalSpacer_16">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="15" column="1">
         <spacer name="verticalSpacer_15">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="21" column="1">
         <spacer name="verticalSpacer_17">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="5">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="16" column="1">
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="13" column="1">
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="10" column="1">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="24" column="1">
         <spacer name="verticalSpacer_18">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leIdwcno</tabstop>
  <tabstop>leIono</tabstop>
  <tabstop>leIdname</tabstop>
  <tabstop>leIdage</tabstop>
  <tabstop>leIdsex</tabstop>
  <tabstop>leIdtitle</tabstop>
  <tabstop>leIdsalary</tabstop>
  <tabstop>btnIDoctor</tabstop>
  <tabstop>btnIClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
